src/lib/utils/SynapseClient.ts

```diff
@@ -832,14 +836,14 @@ export const getUserFavorites = (sessionToken: string | undefined) => {
   ) as Promise<any>
 }
 /**
- *  http://docs.synapse.org/rest/GET/projects/type.html
- *  @param {String} projectDetails Can be "MY_PROJECTS", "MY_CREATED_PROJECTS" or "MY_PARTICIPATED_PROJECTS"
+ *  https://docs.synapse.org/rest/GET/projects.html
+ *  @param {String} projectDetails Can be "CREATED", "PARTICIPATED" or "TEAM"
  */
 export const getUserProjectList = (
```

Comment:
good change, but recommend we just remove these calls since they are not being used today (service does not support offset anymore, for example, so this version certainly doesn't support pagination).